community/blog/2020-03-Releasing-Arcus-Messaging-and-BackgroundJobs-v.0.1.0/README.md

```diff
@@ -0,0 +1,79 @@
+# Announcing Arcus Messaging & Background Jobs
+
+Recently we released v0.1.0 of the [`Arcus.Messaging`](https://github.com/arcus-azure/arcus.messaging/releases/tag/v0.1.0) and [`Arcus.BackgroundJobs`](https://github.com/arcus-azure/arcus.backgroundjobs/releases/tag/v0.1.0) and are now available on NuGet.
+Both help in in writing repeating jobs that securely receive external events and process them any way you want.
+
+Come and take a look at what they have to offer!
+
+## Receiving Messages From Azure Service Bus Queues & Topics
+
+With Arcus Messaging we provide something that's called a **message pump** for messages on Azure Service Bus.
+This pump can be registered very easily on startup of your application.
+
+Once registered, one or more **message handlers** can be registered that will eventually process the message. 
+Based on the message type, the pump will send the message to the correct registered handler.
+This allows a loosely coupled system between receiving and processing.
+
+By using our [`Arcus.Messaging.Pumps.ServiceBus`](https://www.nuget.org/packages/Arcus.Messaging.Pumps.ServiceBus/) NuGet package, you can configure such a pump during startup.
+See the following section as an example to handle `Order` messages with a custom `OrdersMessageHandler` implementation. 
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddServiceBusQueueMessagePump("ServiceBus:Queue:ConnectionString")
+            .WithServiceBusMessageHandler<OrdersMessageHandler, Order>();
+}
+```
+
+Where the your `OrderMessageHandler` will process the `Order` messages received on the Azure Service Bus Queue:
+
+```csharp
+public class OrdersMessageHandler : IMessageHandler<Order>
+{
+    public async Task ProcessMessageAsync(
+        Order message, 
+        MessageContext messageContext, 
+        MessageCorrelationInfo correlationInfo, 
+        CancellationToken cancellationToken)
+    {
+    }
+}
```

Comment:
This example isn't very descriptive as it doesn't do anything?

Reply:
Added logging.

Reply:
Great, thanks!